Dockerfile.arm64

```diff
@@ -0,0 +1,13 @@
+FROM plugins/base:arm64
+
+LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
+  org.label-schema.name="Drone Greenkeeper" \
+  org.label-schema.vendor="Drone.IO Community" \
+  org.label-schema.schema-version="1.0"
+
+RUN apk add --no-cache git nodejs nodejs-npm yarn
+RUN npm install -g npm@latest \
+  npm install -g donny-dont/greenkeeper-lockfile#feat/drone-ci
```

Comment:
Waiting on https://github.com/greenkeeperio/greenkeeper-lockfile/pull/141 to merge before changing the endpoint.